tip/tip.py

```diff
@@ -0,0 +1,199 @@
+#!/usr/bin/env python3
+
+import re
+from sys import argv, stdout, exit as sys_exit
+from os import environ, getenv
+from glob import glob
+from pathlib import Path
+from github import Github
+from github import GithubException
+
+print("· Get list of artifacts to be uploaded")
+
+args = []
+files = []
+
+if "INPUT_FILES" in environ:
+    args = environ["INPUT_FILES"].split()
+
+if len(argv) > 1:
+    args = args + argv[1:]
+
+if len(args) == 1 and args[0] == "none":
+    files = []
+    print("! Skipping 'files' because it's set to 'none")
+else:
```

Comment:
else and following if collapse to an elif.

Then following for can be deindented.

Reply:
I added an `elif`. However, I don't think the `for` can be deindented. If the first condition is met it means that no files are to be uploaded (it's just the creation/update of a release).

Reply:
Actually the "no files" feature is used in GHDL. There, all "nightly" builds do upload the artifacts as assets, but tagged builds/releases do not have assets.

Reply:
OK.